### ISSUE TITLE
how to provide plugins with a config

### ISSUE THREAD

**Issue by arfost** [jsdoc/jsdoc#1574]
Hi,
I'm trying to do a jsDoc to plantUML plugin, and want to add some configs to it.
I wonder if there is a recommended way to get them.

Apparently plugins are not provided with any info from the jsDoc config file and the only solution 
that come to my mind is to add another config file and request it in the plugin, but there is already many in our projects, maybe i'll need more plugins and don't want to multiply them.

In short, I just wanted to know from seasoned users if I missed something

Thanks in advance for all answers :)

**Comment by JustinBeaudry** [comment]
@arfost I know this is almost a year too late, but I thought I'd answer regardless:

From:  https://github.com/trilobyte-berlin/jsdoc-plantuml/blob/master/index.js#L34

In your jsdoc configuration JSON `conf.json` or whatever you call it you can add:
```json
{
  plugins: 'jsdoc-plantuml',
  // plugin specific configuration
  plantuml: {
    // default configuration from https://github.com/trilobyte-berlin/jsdoc-plantuml/blob/master/index.js#L14
    createPuml: true,
    createImages: true,
    pathPuml: './jsDoc/puml',
    pathImages: './jsDoc/images',
    imageFormat: 'png'
  }
}
```
